### ISSUE TITLE
Missing accessible link text on DSS regions page

### ISSUE THREAD

**Issue by vh8154** [pulibrary/DSS#471]
### Steps to recreate
1. Go to https://dss.princeton.edu/regions
2. Log in
3. Visit [this page](https://dss.princeton.edu/regions)
4. click "edit" for one of the rows
5.  click "Update Region".
6. Run axe DevTools on this page.

### URL
https://dss.princeton.edu/regions

### Issues
Links must have discernible text [This page has 4 occurrences of this issue]

### Severity
Serious

### WCAG Success Criterion
[WCAG 2.4.4: Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context)
[WCAG 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value)

### WCAG Conformance level (A, AA, AAA)
A

### User impact
Element does not have accessible text, so users that utilize screen readers will be affected.

### Recommended fix
To solve this problem, you need to fix at least (1) of the following:
- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute

And fix the following:
- Element is in tab order and does not have accessible text

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

**Comment by vh8154** [comment]
From using inspect, I noticed the 4 links are in lux footer.

**Comment by vh8154** [comment]
Blocked by #424
Since these links are in the lux footer, changing to the new footer may solve this issue.

**Comment by vh8154** [comment]
Closed by #424
